concrete/src/Multilingual/Page/Section/Section.php

```diff
@@ -328,9 +328,7 @@ public static function registerDuplicate($newPage, $oldPage)
                 }
             }
 
-            $v = [$mpRelationID, $newPage->getCollectionID(), $ms->getLocale()];
-
-            if (!$isNew) {
+            if ($isNew) {
```

Comment:
This is the only relevant change, isn't it?

Reply:
Yes, the other one is just cosmetic. But makes the code more readable..

Reply:
I don't know.
@1stthomas ?